scripts/check_frontend_coverage.py

```diff
@@ -283,6 +283,7 @@
     'playthrough-issues.directive.ts',
     'Polyfills.ts',
     'practice-tab.directive.ts',
+    'preferences-page.component.ts',
```

Comment:
We don't add more files to the list, you can look at the karma coverage (https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#generating-coverage-reports) to figure out where it is going wrong.

Reply:
Done